addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1339,18 +1340,21 @@ const Wysiwyg = Widget.extend({
         const mediaDialog = new weWidgets.MediaDialog(this, mediaParams, $node);
         mediaDialog.open();
 
-        mediaDialog.on('save', this, function (element) {
+        mediaDialog.on('save', this, async function (element) {
```

Comment:
Could you add a test for this fix please? I have a feeling that it is likely to be broken in the future due to the introduction of asynchronicity in a function that uses the legacy compatibility layer of owl.